### DISCUSSION TITLE
Kiali integrates keycloak authentication, and login users cannot change istio configuration

### DISCUSSION THREAD

**Discussion by nextify2025** [kiali/kiali#6365] [Q&A]
If I log in with keycload, I can't modify the istio config, but if I use the anonymous authentication policy, I can modify it. Is there any way to make the user logged in with keycload also modify the istio configuration?
<img width="1131" alt="image" src="https://github.com/kiali/kiali/assets/10096899/31e86f8c-821a-4262-82dc-034556011633">

**Comment by jmazzitelli** [comment]
It sounds like you may have installed Kiali with view_only_mode enabled.

You did not provide the Kiali version you are using, how you installed it, and what your Kiali CR looks like (if you installed via operator ) or what your Kiali ConfigMap looks like (if you installed via server helm chart).

But have a look at this setting and make sure you have not enabled view_only_mode: https://kiali.io/docs/configuration/kialis.kiali.io/#.spec.deployment.view_only_mode
